client/me/concierge/controller.js

```diff
@@ -74,9 +75,16 @@ const siteSelector = ( context, next ) => {
 	next();
 };
 
+const redirectToQuickStart = ( context, next ) => {
+	const newPath = context.path.replace( '/me/concierge/', '/me/quickstart/' );
```

Comment:
This doesn't work if the site slug isn't provided. e.g. in the following screencast, `concierge` is replaced by `quickstart` as expected when the site slug is provided, but it leads to a blank page without rewriting if the site slug isn't provided:
![rewrite-quickstart](https://user-images.githubusercontent.com/1842898/128144337-6b942ebe-3440-41f5-8b02-d3ceaf503638.gif)

This should be fine, but I'd still like to mention it here since I've bumped into it :)